common/lanelet2_extension/include/lanelet2_extension/logging/logger.h

```diff
@@ -0,0 +1,49 @@
+
+#ifndef LANELET2_EXTENSION_LOGGER_H
+#define LANELET2_EXTENSION_LOGGER_H
+
+/*
+ * Copyright (C) 2020-2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#define LOGGER_TYPE_ROS_LOGGER 0
+#define LOGGER_TYPE_STD_LOGGER 1
+
+// If the logger type is not set then set it to the default
+
+#if LANELET2_EXTENSION_LOGGER_TYPE == LOGGER_TYPE_STD_LOGGER // Logger type of 1 indicates usage of std::cout 
+  
+#include <iostream> // Include std::cout and cerr
+
+#define LOG_DEBUG_STREAM(args) std::cout << args << std::endl;
```

Comment:
I think we might want to add parens around `(args)`?  Just for safety to ensure that any concatenation or other processing in there happens first. I'm uncertain of the associativity and order of operations applied w.r.t. the `<<` stream operator but if your confident that there's no need for concern then that's fine too. 

Also just a bit weird that this case has a semicolon and the other doesnt, but that shouldn't functionally matter.

Reply:
Actually this is more likely to cause issues then not having it. 
If you add () then the following is no longer able to compile

LOG_DEBUG_STREAM("hello" << my_var)

This is because the leading string does not resolve to a stream operator, so I would opt to leave the () out so it can work like the ros loggers.